internal/core/Cargo.toml

```diff
@@ -83,6 +85,8 @@ resvg = { version= "0.23", optional = true, default-features = false }
 usvg = { version= "0.23", optional = true, default-features = false, features = ["text"] }
 tiny-skia = { version= "0.6", optional = true, default-features = false }
 
+copypasta = { version = "0.8.1", optional = true, default-features = false, features = ["x11"] }
```

Comment:
oh, x11 and wayland again....

Reply:
Should be fine I think. We don’t use the Wayland but and the x11 features compiles also on non-x11